prow/cmd/deck/static/script.js

```diff
@@ -418,25 +422,50 @@ function redraw(fz) {
     redrawOptions(fz, opts);
 
     var lastKey = '';
-    for (var i = 0, emitted = 0; i < allBuilds.length && emitted < 500; i++) {
-        var build = allBuilds[i];
-        if (!equalSelected(typeSel, build.type)) {
+    for (var i = 0, emitted = 0; i < allBuilds.items.length && emitted < 500; i++) {
+        var build = allBuilds.items[i];
+
+        // assign default values 
+        var prop = { 
```

Comment:
I would skip this, and instead change the one case that uses build.spec.refs to skip if build.spec.refs is empty.